src/ChildProcessUtilities.js

```diff
@@ -1,4 +1,5 @@
 import child from "child_process";
+import objectAssign from "object-assign";
```

Comment:
Isn't it possible to use Object.assign directly? (I'm not sure whether babel polyfills it or not, to be honest)